src/CommunityToolkit.Mvvm.SourceGenerators/ComponentModel/Models/PropertyInfo.cs

```diff
@@ -30,5 +31,6 @@ internal sealed record PropertyInfo(
     bool NotifyPropertyChangedRecipients,
     bool NotifyDataErrorInfo,
     bool IsOldPropertyValueDirectlyReferenced,
-    bool IsReferenceType,
+    bool IsReferenceTypeOrUnconstraindTypeParameter,
+    bool IncludeMemberNotNullOnSetAccessor,
```

Comment:
With this many `bool` properties, I'd recommend switching to a bitmask personally. Especially if you ever plan to add unit testing that will construct this with specific values (instead of always passing in well-named variables as you do today).